### PR TITLE
Admin Kicks now prompt the admin to avoid accidental kicking

### PR DESCRIPTION
**Pull request by ghost**
@optimumtact here ya go buddy!

why? accidental kicks stay with an admin all day

### CODE REVIEW

**Comment by CitrusGender** [comment]
no arm, oranges is going to complain about your git commits!

**Comment by Armhulen** [comment]
stupid broken computer clock ruining my commit jokes ugh

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -1060,11 +1060,21 @@
 			if(!check_if_greater_rights_than(M.client))
 				to_chat(usr, "<span class='danger'>Error: They have more rights than you do.</span>")
 				return
-			to_chat(M, "<span class='danger'>You have been kicked from the server by [usr.client.holder.fakekey ? "an Administrator" : "[usr.client.ckey]"].</span>")
-			log_admin("[key_name(usr)] kicked [key_name(M)].")
-			message_admins("<span class='adminnotice'>[key_name_admin(usr)] kicked [key_name_admin(M)].</span>")
-			//M.client = null
-			qdel(M.client)
+			switch(alert(usr, "Kick [key_name(M)]?", "CHATSHITGETBANGED", "Yes", "No"))
+				if("Yes")
```
Comment (ninjanomnom) [inline code/modules/admin/topic.dm]:
```
if(alert(usr, "Kick [key_name(M)]?", "CHATSHITGETBANGED", "Yes", "No") == "No")
  return
...
```

**Review by ninjanomnom — DISMISSED** [review]

File: code/modules/admin/topic.dm [hunk 2]
```diff
@@ -1060,11 +1060,21 @@
...
+					if(!M)
+						to_chat(usr, "<span class='danger'>Error: [M] no longer exists!</span>")
+						return
+					if(!M.client)
+						to_chat(usr, "<span class='danger'>Error: [M] no longer has a client!</span>")
+						return
+					to_chat(M, "<span class='danger'>You have been kicked from the server by [usr.client.holder.fakekey ? "an Administrator" : "[usr.client.ckey]"].</span>")
+					log_admin("[key_name(usr)] kicked [key_name(M)].")
+					message_admins("<span class='adminnotice'>[key_name_admin(usr)] kicked [key_name_admin(M)].</span>")
+					//M.client = null
```
Comment (Jordie0608) [inline code/modules/admin/topic.dm]: Remove this line.

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (Armhulen) [reply code/modules/admin/topic.dm]: cool trick, i'll add it

**Comment by Armhulen** [comment]
@ninjanomnom as you asked!

**Comment by Cyberboss** [comment]
Fix your shit

File: code/modules/admin/topic.dm [hunk 3]
```diff
@@ -1056,14 +1056,21 @@
 		if(!check_rights(R_ADMIN))
 			return
 		var/mob/M = locate(href_list["boot2"])
-		if (ismob(M))
+		if(ismob(M))
 			if(!check_if_greater_rights_than(M.client))
 				to_chat(usr, "<span class='danger'>Error: They have more rights than you do.</span>")
 				return
+			if(alert(usr, "Kick [key_name(M)]?", "CHATSHITGETBANGED", "Yes", "No") != "Yes")
```
Comment (Cyberboss) [inline code/modules/admin/topic.dm]: Come the fuck on, man

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/admin/topic.dm (hunk 3 above)
Comment (Armhulen) [reply code/modules/admin/topic.dm]: What's wrong

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**